uvm/impl/com/untangle/uvm/AppManagerImpl.java

```diff
@@ -1455,6 +1510,35 @@ private AppManagerSettings loadSettings()
         return this.settings;
     }
 
+    /**
+     * Sync licenses with apps if restricted
+     */
+    public void syncWithLicenses() {
+        AppManagerImpl nm = (AppManagerImpl) UvmContextFactory.context().appManager();
+        LicenseManager lm = UvmContextFactory.context().licenseManager();
+        Map<String, License> lmLicenses = new HashMap<>();
+
+        if (!lm.isRestricted()) {
+            logger.info("Not restricted");
+            return;
+        } 
+
+        logger.info("Restricted");
+        for(License lic : lm.getLicenses()) {
+            String licenseName = fixupName(lic.getCurrentName());
```

Comment:
Why do we have to call fixupName before adding licenses into the license manager (each of the lmLicenses.put calls)? is this something that getCurrentName() should be doing automatically?